studygroups/models.py

```diff
@@ -177,9 +177,9 @@ class StudyGroup(LifeTimeTrackingModel):
     venue_details = models.CharField(max_length=128)
     venue_website = models.URLField(blank=True)
     city = models.CharField(max_length=256)
-    latitude = models.DecimalField(max_digits=8, decimal_places=6, null=True)
-    longitude = models.DecimalField(max_digits=9, decimal_places=6, null=True)
-    place_id = models.CharField(max_length=256)
+    latitude = models.DecimalField(max_digits=8, decimal_places=6, null=True, blank=True)
+    longitude = models.DecimalField(max_digits=9, decimal_places=6, null=True, blank=True)
+    place_id = models.CharField(max_length=256, null=True, blank=True)
```

Comment:
For a CharField you should only set `blank=True` and not `null=True` for an optional field. Setting both implies that `None` and `''` (an empty string) could represent blank values,